pkg/graph/resolvers/trb_request_test.go

```diff
@@ -1,98 +1,180 @@
 package resolvers
 
 import (
+	"context"
+	"time"
+
 	"github.com/cmsgov/easi-app/pkg/models"
 )
 
 // TestCreateTRBRequest makes a new TRB request
-func (suite *ResolverSuite) TestCreateTRBRequest() {
-
+func (s *ResolverSuite) TestCreateTRBRequest() {
 	//TODO get the context in the test configs
-	trb, err := CreateTRBRequest(suite.testConfigs.Context, models.TRBTBrainstorm, suite.testConfigs.Store)
-	suite.NoError(err)
-	suite.NotNil(trb)
-
-	suite.EqualValues(trb.Archived, false)
-	suite.EqualValues(trb.Name, "Draft")
-	suite.EqualValues(trb.Status, models.TRBSOpen)
-	suite.EqualValues(trb.CreatedBy, suite.testConfigs.Principal.EUAID)
-	suite.NotNil(trb.ID)
-	suite.NotNil(trb.CreatedAt)
-	suite.Nil(trb.ModifiedBy)
-	suite.Nil(trb.ModifiedAt)
-
+	trb, err := CreateTRBRequest(s.testConfigs.Context, models.TRBTBrainstorm, s.testConfigs.Store)
+	s.NoError(err)
+	s.NotNil(trb)
+
+	s.EqualValues(trb.Archived, false)
+	s.EqualValues(trb.Name, "Draft")
+	s.EqualValues(trb.Status, models.TRBSOpen)
+	s.EqualValues(trb.CreatedBy, s.testConfigs.Principal.EUAID)
```

Comment:
These assertions are flipped around; if you look at the Intellisense, the expected value goes first, followed by the value being tested.